doc/internal/ReleaseInstructions.md

```diff
@@ -151,218 +85,195 @@ Therefore, file a JIRA ticket with Sonatype to get added ([example for a differe
+    ```shell
+    git checkout v12.0.0
     ```
 
-1.  Run `git diff java/` to double check that the version was updated correctly.
-
-1.  Create a Git commit with the version change while you are in the **release** branch (not main). You will push it later.
-
-1.  Run `gpg-agent` to avoid that Maven will constantly prompt you for the password of your private key.
+2.  Run `gpg-agent` to avoid that Maven will constantly prompt you for the password of your private key.
 
     ```bash
     eval $(gpg-agent --daemon --no-grab --write-env-file $HOME/.gpg-agent-info)
     export GPG_TTY=$(tty)
     export GPG_AGENT_INFO
     ```
 
-1.  Deploy (upload) the Java code to the oss.sonatype.org repository:
+3.  Deploy (upload) the Java code to the oss.sonatype.org repository:
 
-    <p class="warning"><b>Warning:</b> After the deployment, the Java packages will be automatically released. Once released, you cannot delete them. The only option is to upload a newer version (e.g. increment the patch level).</p>
+    > **Warning:** After the deployment, the Java packages will be automatically released. Once released, you cannot delete them. The only option is to upload a newer version (e.g. increment the patch level).</p>
 
     ```bash
-    mvn clean deploy -P release
+    mvn clean deploy -P release -DskipTests
     cd ..
     ```
-
-At the end of the release, you will also have to bump the SNAPSHOT version in the main branch (see [below](#bump-java-snapshot-version)).
-
-## Push the release branch and tag to upstream
-
-Note that we're pushing to upstream (vitessio/vitess), not origin (your fork).
-
-<p class="warning"><b>Warning:</b> After the following push, there's no going
-back, since tags don't get updated if someone else has fetched them already.
-If you need to re-tag after this point, you MUST increment the version number.
-</p>
-
-```bash
-# release branch
-git push upstream release-X.Y
-# release tag
-git push upstream vX.Y.Z
-```
-
-## Add release notes and send announcement
-
-[Find your new tag](https://github.com/vitessio/vitess/tags) and add release
-notes. Use the GitHub [Compare](https://github.com/vitessio/vitess/compare) tool
-to see all the commits since the last release.
-
-Then send an announcement on the [vitess-announce](https://groups.google.com/forum/#!forum/vitess-announce) list.
-
-## Bump Java SNAPSHOT version
-
-The Java version in the **main** branch has the suffix `-SNAPSHOT` and should always reference the **next** release number.
-For example, if you just released `2.1.0`, you could bump it to `2.1.1-SNAPSHOT`.
-
-Change the SNAPSHOT version as follows:
-
-```bash
-git checkout main
-git pull --ff-only upstream main
-
-# Example:
-# (Note that there is no leading "v".)
-JAVA_VERSION=2.1.1-SNAPSHOT
-cd java
-mvn versions:set -DnewVersion=$JAVA_VERSION
-
-# Verify diff.
-git diff
-
-git commit -a -m "java: Bump SNAPSHOT version to $JAVA_VERSION after Vitess release $PATCH."
-
-git push upstream main
-```
```

Comment:
This was removed as it is automated through the `release_notes`and `do_release` Makefile commands.